src/core/BufferAttribute.js

```diff
@@ -86,6 +86,27 @@ class BufferAttribute {
 
 	}
 
+	fill( arrayLike ) {
+
+		const end = Math.min( arrayLike.length, this.array.length );
+
+		for ( let i = 0; i < end; i ++ ) {
+
+			this.array[ i ] = arrayLike[ i ];
+
+		}
+
+		// copy twice the amount of previosu iteration
+		for ( let i = end, il = this.array.length; i < il; i <<= 1 ) {
```

Comment:
```
i <<= 1
```
TBH, I find this bitwise operation always hard to read. Can this be written in a different way?

Reply:
ok.